src/views/Correction.vue

```diff
@@ -584,6 +588,20 @@ export default class Correction extends Mixins(CommonMixin, DateMixin, EnumMixin
     // prevent double saving
     if (this.busySaving) return
 
+    if (!this.isPageValid) {
+      this.showErrors = true
+
+      // Show error messages of components if invalid
+      if (!this.detailCommentValid) {
+        this.$refs.detailCommentRef.$refs.textarea.error = true
```

Comment:
This double-deep refs is not good practice. (It's bad enough to reach into 1 component to set or call something.)

I think it's also bad practice to set a component's error to True. Instead, it would be better to assign some "rules" to that component and then having the component check them (eg, by calling `validate()` on it, if needed).

Reply:
The Detail Comment component may also accept a prop to ask it to validate itself.

Reply:
This code was re-written and this conversation is obsolete. Resolving.